cli/bpmetadata/tfconfig.go

```diff
@@ -441,9 +441,36 @@ func parseBlueprintRoles(rolesFile *hcl.File) ([]*BlueprintRoles, error) {
 		break
 	}
 
+	sortBlueprintRoles(r)
 	return r, nil
 }
 
+// Sort blueprint roles.
+func sortBlueprintRoles(r []*BlueprintRoles) {
+	sort.SliceStable(r, func(i, j int) bool {
```

Comment:
Let's add a unit test for this exercising all conditions so incase something changes in the future we can avoid spurious diffs across our repos.

Reply:
Added UTs, PTAL!